machine-setup/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:16.04
+
+ADD *.py ./
+RUN apt-get update
+RUN apt-get install -y openssh-client
```

Comment:
Put in one line like `openssh-client python curl` etc, then Docker won't generate image for each time. In fact you can do `apt-get update && apt-get install -y xxxx`